net/net.sh

```diff
@@ -61,7 +61,7 @@ Operate a configured testnet
                                       - Override the default --hashes-per-tick for the cluster
    --no-airdrop
                                       - If set, disables airdrops.  Nodes must be funded in genesis config when airdrops are disabled.
-   --lamports NUM_LAMPORTS_TO_MINT
+   --mint-lamports NUM_LAMPORTS_TO_MINT
                                       - Override the default 500000000000000000 lamports minted in genesis
```

Comment:
How about a default of zero?

Reply:
Without the `--mint-pubkey` present the default doesn't really matter.   But making the default zero means that you now much always supply `--mint-lamports` whenever `--mint-pubkey` is provided.

Reply:
It'll affect the output of `capitalization()`.

Reply:
https://github.com/solana-labs/solana/blob/cc96848b016663b2010a845ea8d7f620a5f361a7/runtime/src/bank.rs#L527

Reply:
Why is that?

Reply:
No.  The lamports will not exist if `--mint-pubkey` is not provided

Reply:
Got it, thanks.

Reply:
But the default is now nothing too, `--faucet-lamports` is now optional